src/main/java/org/usfirst/frc/team5687/robot/subsystems/Intake.java

```diff
@@ -71,16 +71,13 @@ public boolean isPrimeAble() {
     public void updateDashboard() {
         SmartDashboard.putNumber("IR distance", boulderSensor.getValue());
         if (!isDetected()){
-            SmartDashboard.putString("Boulder is", "Not Detected");
-        }
-        else if (!isPrimeAble()){
-            SmartDashboard.putString("Boulder is", "Detected");
```

Comment:
@rtfoley yes, it doesn't need to be there, right? Or could the condition be isDetected() instead?